setup.cfg

```diff
@@ -7,7 +7,7 @@ testpaths = tests
 universal=1
 
 [flake8]
-max-line-length = 99
+max-line-length = 119
```

Comment:
Modified this to be consistent with the Django max line convention. Ref.: https://github.com/django/django/blob/master/setup.cfg#L8